cmd/gardener-resource-manager/app/app.go

```diff
@@ -83,6 +83,10 @@ func NewResourceManagerCommand() *cobra.Command {
 			targetClientOpts.Completed().Apply(&healthControllerOpts.Completed().TargetClientConfig)
 			resourceControllerOpts.Completed().ApplyClassFilter(&secretControllerOpts.Completed().ClassFilter)
 			resourceControllerOpts.Completed().ApplyClassFilter(&healthControllerOpts.Completed().ClassFilter)
+			err := resourceControllerOpts.Completed().ApplyDefaultClusterId(entryLog, sourceClientOpts.Completed().RESTConfig)
```

Comment:
```suggestion
			err := resourceControllerOpts.Completed().ApplyDefaultClusterId(cmd.Context(), entryLog, sourceClientOpts.Completed().RESTConfig)
```

Reply:
I'll use the local ctx that got a cancel function

Reply:
Why not the command's context?